docs/source/quick_start/install.rst

```diff
@@ -0,0 +1,199 @@
+
+Installation
+============
+
+For the first step of our journey, let's install MCDR
+
+.. note::
+
+    We assume you have some really basic knowledge of Python and pip.
+
+    If you have not, Google is your friend
+
+Requirements
+------------
+
+.. image:: https://img.shields.io/pypi/pyversions/mcdreforged.svg
+   :alt: Python version
+
+.. image:: https://img.shields.io/pypi/v/mcdreforged.svg
+   :alt: PyPI version
+
+MCDR is written and runs in Python 3. Make sure you have Python 3.8 or later installed
+
+The detailed Python version requirements are as shown in the table below
+
+.. list-table::
+   :header-rows: 1
+
+   * - MCDR version
+     - Python requirement
+   * - < 2.10
+     - >= 3.6
+   * - >= 2.10
+     - >= 3.8
+
+Install using pip...?
+----------------------
+
+MCDR is available in `PyPI <https://pypi.org/project/mcdreforged>`__, which means it can be installed via the ``pip`` command:
+
+.. code-block:: bash
+
+    pip install mcdreforged
+
+Externally managed environment
+------------------------------
+
+If you're using Windows, the command above should work fine, MCDR will be installed to global environment - you may ignore this section
+
+However, on Linux and Mac OS, if you're using a newer version of Python, you'll get something like this when you run pip install:
+
+.. code-block:: bash
+    
+    $ pip install mcdreforged
+    error: externally-managed-environment
+
+    × This environment is externally managed
```

Comment:
This output only exists in a few unix distros, and is a bit too long to be put here. I'll suggest to replace it with explaination + pep link instead, e.g.

```rst
For Linux and Mac OS, it's not recommended to install MCDR system-wide
because it can cause conflicts with other Python packages and affect system dependencies. 
Installing it this way also makes version management difficult 
and requires administrator privileges, increasing security risks.
Using a virtual environment etc. is safer as it keeps the installation isolated to specific projects

See `PEP 668 <https://peps.python.org/pep-0668/>`__ for the detailed specification
```